### PR TITLE
NetCDF checker - python call function

### PR DESCRIPTION
**Pull request by lbesnard**
* move function from aims_realtime to util.py. more common lib
 * add unittests

### CODE REVIEW

**Comment by lbesnard** [comment]
tested on NSP14, as cannot be tested on PO box since the nc checker is not up to date

**Comment by lwgordonimos** [comment]
@lbesnard For an existing PO box, the following should get rid of the old plugin links and switch it over to the new packaged version:

```
sudo rm -f /usr/local/lib/python2.7/dist-packages/{easy-install.pth,cc-plugin-imos.egg-link}
sudo easy_install --always-unzip /var/chef/cache/cc_plugin_imos-0.9.0-py2.7.egg
```

**Comment by lbesnard** [comment]
Thx, works well on PO

File: lib/test/python/test_nc_checker_call.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import unittest
+
+class TestPassChecker(unittest.TestCase):
+
+    def setUp(self):
+        self.nc_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'test.nc')
+
+    def test_pass_netcdf_checker(self):
+        self.assertTrue(pass_netcdf_checker(self.nc_path, ['cf:1.6']))
+        self.assertTrue(pass_netcdf_checker(self.nc_path, ['imos:1.3']))
```
Comment (lbesnard) [inline lib/test/python/test_nc_checker_call.py]: @mhidas Are we going to implement proper versionning of the IMOS checker such as the ACDD one ?

Comment (mhidas) [inline lib/test/python/test_nc_checker_call.py]: It's already done, i.e. what you have there should work. For now we only have 'imos:1.3', but eventually we'll have 'imos:1.4' as well. 'imos' and 'imos:latest will default to the latest version of the conventions.
(Run `compliance-checker -l` to see all available check suites.)

**Comment by lbesnard** [comment]
@lwgordonimos , see consol ouput for data services prod merge

```
======================================================================
ERROR: test_pass_netcdf_checker (__main__.TestPassChecker)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "lib/test/python/test_nc_checker_call.py", line 18, in test_pass_netcdf_checker
    self.assertTrue(pass_netcdf_checker(self.nc_path, ['cf:1.6']))
  File "/home/jenkins/slave/workspace/data_services_prod_merge_test_unit/lib/python/util.py", line 114, in pass_netcdf_checker
    from compliance_checker.runner import ComplianceChecker, CheckSuite
ImportError: No module named compliance_checker.runner
```

**Comment by lwgordonimos** [comment]
@lbesnard I have a PR up for this, so should be resolved as soon as one of the others guys merges it

**Comment by mhidas** [comment]
We could make this function (and its unittest) part of the cc_plugin_imos package?

**Comment by lbesnard** [comment]
sounds reasonable

**Comment by lwgordonimos** [comment]
@mhidas the unit test(s) definitely, but I can see value to keeping the utilty function(s) separate, since they are consumers of the plugin, not part of it.

**Comment by mhidas** [comment]
@lwgordonimos That would mean the unittest in cc_plugin_imos would be dependent on a function that is outside it... is that wise?

**Pull request merged**